mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewclient/CreateNewClientFragment.java

```diff
@@ -232,6 +234,20 @@ public void showUserInterface() {
         tvSubmissionDate.setText(MFDatePicker.getDatePickedAsString());
         tvDateOfBirth.setText(MFDatePicker.getDatePickedAsString());
 
+        etClientExternalId.setOnEditorActionListener(new TextView.OnEditorActionListener() {
+            @Override
+            public boolean onEditorAction(TextView textView, int i, KeyEvent keyEvent) {
+                if(i == EditorInfo.IME_ACTION_DONE){
+                    hideKeyboard(etClientExternalId);
+                    etClientExternalId.clearFocus();
+                    etClientExternalId.setCursorVisible(false);
+                    return true;
+                }
+                return false;
+            }
+        });
+
+
```

Comment:
@Abhinavtdk if possible please remove unnecesary empty lines.

Reply:
Yea, I'll change it